spring-cloud-dataflow-server-cloudfoundry-autoconfig/src/main/java/org/springframework/cloud/dataflow/server/cloudfoundry/config/DataSourceCloudConfig.java

```diff
@@ -34,12 +38,19 @@
 public class DataSourceCloudConfig extends AbstractCloudConfig {
 
 	@Bean
-	public DataSource dataSource(CloudFoundryServerConfigurationProperties cloudFoundryServerConfigurationProperties) {
-		PooledServiceConnectorConfig.PoolConfig poolConfig =
-				new PooledServiceConnectorConfig.PoolConfig(cloudFoundryServerConfigurationProperties.getMaxPoolSize(),
-						cloudFoundryServerConfigurationProperties.getMaxWaitTime());
+	public DataSource scdfCloudDataSource(
```

Comment:
Why changing bean name?

Reply:
I had an implementation where I had to delete an extra datasource bean that either the connector library or autoconfiguration so I had a bean factory post processor to zap the other one and wanted this one to be kept, to made a crazy name.